.github/workflows/tests.yml

```diff
@@ -14,11 +14,13 @@ jobs:
       id: set-up-homebrew
       run: |
         if which brew &>/dev/null; then
-          HOMEBREW_REPOSITORY="$(brew --repo)"
+          HOMEBREW_PREFIX="$(brew --prefix)"
+          HOMEBREW_REPOSITORY="$HOMEBREW_PREFIX/Homebrew"
           brew update-reset "$HOMEBREW_REPOSITORY"
 
           HOMEBREW_CORE_REPOSITORY="$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-core"
-          ln -s "$PWD" "$HOMEBREW_CORE_REPOSITORY"
+          git -C "$HOMEBREW_CORE_REPOSITORY" remote set-url origin "https://github.com/$GITHUB_REPOSITORY"
```

Comment:
Could we use `[HOMEBREW_USE_HOMEBREW_ON_LINUX=1](https://github.com/Homebrew/brew/blob/fa980e9c427b41db245a15c4bbcd510963b52169/Library/Homebrew/brew.sh#L408-L413)` instead of hand-rolling this?

Reply:
`HOMEBREW_FORCE_HOMEBREW_ON_LINUX` doesn't appear to change the existing git origin of the core tap.

Reply:
Oh. As you were!